digitalpublicgoods/mhero.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "mHero",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://www.mhero.org/faq#resources"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://github.com/intrahealth/emNutt"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "Yes",
+    "checkNonPIIAccessMechanism": "Yes",
+    "nonPIIAccessMechanism": "exportation of data is possible and all reports include PII (the PII could be removed from the exported reports)"
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Yes",
+    "privacyComplianceList": [
+      "Various laws at the country level everywhere mHero is implemented"
+    ],
+    "adherenceSteps": [
+      "Local laws about data storage are reviewed determine how and where data is stored. Data sharing agreements are established so that the core software developers can access system to perform diagnosis of issues, fixes, updates, etc. (https://www.mhero.org/sites/mhero/files/sampledatasharingenglish.docx ; https://www.mhero.org/sites/mhero/files/sampledatasharingfrench.docx)"
+    ]
+  },
+  "standards": {
+    "supportStandards": "Yes",
+    "standardsList": [
+      "FHIR",
+      "mACM"
+    ],
+    "evidenceStandardSupport": [
+      "https://github.com/intrahealth/emNutt",
+      "https://github.com/iHRIS/iHRIS"
```

Comment:
Without additional context, this link seems irrelevant as it belongs to an entirely different project.